### PR TITLE
Change crdApplied to required for google marketplace

### PR DESCRIPTION
**Pull request by waodim**

### CODE REVIEW

**Review by gkrenn — CHANGES_REQUESTED** [review]
Have you tested this? - For me the install script does not seem to enforce setting the enum.

Btw. please update `upload_test.sh` - The path to the crd does no longer exists => Should be `kubectl apply -k https://github.com/Dynatrace/dynatrace-operator/config/crd` now

File: config/helm/schema.yaml [hunk 1]
```diff
@@ -169,10 +169,12 @@ properties:
     x-google-marketplace:
       type: MASKED_FIELD
   crdApplied:
-    type: boolean
-    description: "Please make sure you have applied the latest CRD by following the instructions in the Github Repository."
-    title: "CRD manually applied"
-    default: false
+    type: string
+    description: Please make sure you have applied the latest CRD by following the instructions in the Github Repository.
```
Comment (meik99) [inline config/helm/schema.yaml:173]:
```suggestion
    description: Make sure you have applied the latest CRD by following the instructions in the Github Repository.
```

Comment (meik99) [reply config/helm/schema.yaml:173]: Just a nitpick from my side

**Comment by 0sewa0** [comment]
there is a conflict according to github

**Review by 0sewa0 — APPROVED** [review]

**Pull request merged**